### ISSUE TITLE
Unable to determine Swift version on IOS Platform

### ISSUE THREAD

**Issue by javipacheco** [flame-engine/play_games#6]
I'm using "play_games" lib on my game and the library works fine on Android, but I have a problem when I compile the app on iOS. The error is the next:

```
[!] Unable to determine Swift version for the following pods:

    - `play_games` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the  author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.
```
In my code, I don't show play_games for IOS users, but I need that the library loads in iOS Platform.

I have tried for removing the `ios` folder and moving the code to my project, and it's not possible to use the library on iOS

I don't want to use the library on iOS, I only want to load it

**Comment by luanpotter** [comment]
Hi, this lib doesn't work for iOS! Please refer to #1 (very first issue :)

I believe we must implement an empty swift project or something like that just to handle the dart calls and do nothing.

However, since I don't have access to any iDevices, I would not be able to test and therefore decided not even to try to do that.

Would you like to open a PR? Probably the flutter docs explain the basic setup for swift for flutter plugins. Should be relatively straightforward.

**Comment by erickzanardo** [comment]
@luanpotter Do you know if there is a way for a package to support only one platform? If so I think we should get rid of the ios from this package and implemente Game Center stuff on a new one.

What do you think?

**Comment by javipacheco** [comment]
I have been looking for information about supporting only one platform, and I don't know if it's possible. I think to implement an empty project for ios it's the best idea at this moment.

I'm working on it. I started last night a new repository with an empty project for ios. I'll try it this afternoon and tell you if it worked.

**Comment by javipacheco** [comment]
Hi. I have created a new repository with an empty project for ios

https://github.com/javipacheco/play_games

You can prove it that way.

```
  play_games:
    git:
      url: https://github.com/javipacheco/play_games.git
      ref: master
```

I have tried on my app and works fine. 

Would you like me to create a new PR here with the changes?

**Comment by erickzanardo** [comment]
Sure, a PR would be much appreciated.

**Comment by luanpotter** [comment]
Fixed by #14
